handlers/yo/handler_test.go

```diff
@@ -56,46 +58,106 @@ func setSendURL(s *httptest.Server, h courier.ChannelHandler, c courier.Channel,
 var getSendTestCases = []OutgoingTestCase{
 	{Label: "Plain Send",
 		MsgText: "Simple Message", MsgURN: "tel:+250788383383",
-		ExpectedMsgStatus: "W",
-		MockResponseBody:  "ybs_autocreate_status=OK", MockResponseStatus: 200,
-		ExpectedURLParams: map[string]string{
-			"sms_content":  "Simple Message",
-			"destinations": "250788383383",
-			"ybsacctno":    "yo-username",
-			"password":     "yo-password",
-			"origin":       "2020"},
+		MockResponses: map[string][]*httpx.MockResponse{
+			"*": {
+				httpx.NewMockResponse(200, nil, []byte(`ybs_autocreate_status=OK`)),
+			},
+		},
+		ExpectedRequests: []ExpectedRequest{{Params: url.Values{"sms_content": {"Simple Message"},
+			"destinations": {"250788383383"},
+			"ybsacctno":    {"yo-username"},
+			"password":     {"yo-password"},
+			"origin":       {"2020"},
+		}}},
```

Comment:
if you're rewriting all the tests to use `MockResponses`, you can get rid of `SendPrep` and put the real URL instead of `"*"`

Reply:
I think for URL params the mocks are expecting the entire URL including all the parameters,
I will check how to adjust that to match the URL ignoring the params

Reply:
for now I append * to the URLs

Reply:
makes sense.. you're already testing that the correct params were used in expected requests